x/ecocredit/server/utils/test_utils.go

```diff
@@ -0,0 +1,43 @@
+package utils
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/golang/mock/gomock"
+
+	"github.com/regen-network/regen-ledger/x/ecocredit/core"
+	"github.com/regen-network/regen-ledger/x/ecocredit/mocks"
+)
+
+// ExpectParamGet is a helper function that sets up an expected mock call for the provided type.
+// Once we switch to Go 1.18+ we can switch this impl to be generic:
+// func ExpectParamGet[T any](obj *T, paramKeeper *mocks.MockParamKeeper, times int) {
+//	gmAny := gomock.Any()
+//	paramKeeper.EXPECT().Get(gmAny, gmAny, gmAny).Do(func(_, _ any, param *T) {
+//		*param = *obj
+//	}).AnyTimes()
+// }
+func ExpectParamGet(obj interface{}, paramKeeper *mocks.MockParamKeeper, key []byte, times int) {
```

Comment:
`times` is no longer used?

Reply:
whoops, forgot to add that back https://github.com/regen-network/regen-ledger/pull/1064/commits/867198fc02a6b33dd9c6ee50c56e7aee26adf100